apis/capabilities/v1alpha1/tenant_types.go

```diff
@@ -50,6 +50,17 @@ type TenantSpec struct {
 	TenantSecretRef        v1.SecretReference `json:"tenantSecretRef"`
 	PasswordCredentialsRef v1.SecretReference `json:"passwordCredentialsRef"`
 	MasterCredentialsRef   v1.SecretReference `json:"masterCredentialsRef"`
+	// additional parameters, used for Update, as in master portal Api Docs
+	// +optional
+	FromEmail *string `json:"fromEmail,omitempty"`
+	// +optional
+	SupportEmail *string `json:"supportEmail,omitempty"`
+	// +optional
+	FinanceSupportEmail *string `json:"financeSupportEmail,omitempty"`
+	// +optional
+	SiteAccessCode *string `json:"siteAccessCode,omitempty"`
+	// +optional
+	StateEvent *string `json:"stateEvent,omitempty"`
```

Comment:
`string` type? shouldn't be restricted to limited set of values?

Reply:
Can you test switching from one state (active) to another (suspended) and then back to active. If there are more states, it would be good to test transitions. If I recall, that was not obvious and there were some constraints.

Reply:
we decided meanwhile not add StateEvent, so I removed it. Thank you for comment and explanation.